Library/Formula/pythonpy.rb

```diff
@@ -0,0 +1,58 @@
+class Pythonpy < Formula
+  homepage "https://github.com/Russell91/pythonpy"
+  url "https://pypi.python.org/packages/source/p/pythonpy/pythonpy-0.3.6.tar.gz"
+  sha1 "180e4cd00f89cc00d8a657fc7a2d8af4da8b487c"
+  head "https://github.com/Russell91/pythonpy.git"
+
+  option "without-python", "Build without python 2 support"
+  depends_on :python => :recommended if MacOS.version <= :snow_leopard
```

Comment:
drop the `if...` and the `option` so that it always uses the python in PATH; without a :python dep it'll use system python